R/plotCCA.R

```diff
@@ -225,6 +228,9 @@ setMethod("plotRDA", signature = c(object = "SingleCellExperiment"),
         if ( !is.numeric(ellipse.linewidth) && ellipse.linewidth > 0 ) {
             stop("'ellipse.linewidth' must be a positive number.", call. = FALSE)
         }
+        if( confidence.level < 0 || confidence.level > 1 ){
+          stop("'confidence.level' must be a number between 0 and 1.", call. = FALSE)
+        }
```

Comment:
Ahh sorry, I did not send the comment. Can you add is.numeric()? That will ensure that TRUE and FALSE will be not passed through